src/main/java/com/example/a1project/controllers/MainScreenController.java

```diff
@@ -1,16 +1,145 @@
 package com.example.a1project.controllers;
 
 import java.io.IOException;
+import java.net.URL;
+import java.util.ResourceBundle;
 
+import com.example.a1project.Task;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
 import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.geometry.Insets;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.GridPane;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
 
-public class MainScreenController extends SceneController {
+public class MainScreenController extends SceneController implements Initializable {
+	
+    @FXML
+    private GridPane currentTaskGridPane;
 
+    @FXML
+    private GridPane completedTaskGridPane;
+    
+    ObservableList<Task> currentTasks = FXCollections.observableArrayList();
+	ObservableList<Task> completedTasks = FXCollections.observableArrayList();
+    
 	public void addTask(ActionEvent event) throws IOException {
 		showNewTaskPopup(event);
 	}
 	
 	public void showAllTasks() {
 		
 	}
+	
+	private void newCurrentTaskDisplay(Task task) {
+		
+		AnchorPane anchor = setupTaskItem(task);
+		
+		currentTaskGridPane.add(anchor, 1, currentTaskGridPane.getRowCount());
```

Comment:
I'm not too familiar with this portion but @jzho987 would know! Also wondering whether setting .getRowCount() as the position is evident enough or if we should add a comment. I think it is more beneficial if we do

Reply:
The 1 refers to the column that the AnchorPane is added to, but since the GridPane we created only has one column, we can just change it to 0 and it would be the same effect.
I've added a comment to clarify what the value means at both occurrences of this.